### PR TITLE
Ghost QoL: Check plant genes and reagents easily

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request

See title. Now you can check pill/patch/anything's reagents with chem scan enabled. Examining seeds/trays/grafts will now show all of their traits to you as a ghost.

## Why It's Good For The Game

It makes easier to find about peoples' gimmicks as a ghost

## Changelog
:cl:
qol: You can now check plant genes as a ghost
qol: Chem scan now allows to check pill/patch/whatever reagents as a ghost
/:cl:

### CODE REVIEW

**Comment by MrMelbert** [comment]
I'm a fan of this change, but I have a feeling there mayyyy be a better way of going about it than copy+pasting all the analyzer procs - I dunno, seems a little messy.

**Comment by SmArtKar** [comment]
> I'm a fan of this change, but I have a feeling there mayyyy be a better way of going about it than copy+pasting all the analyzer procs - I dunno, seems a little messy.

spawning an analyser, making it scan the plant, deleting it and then giving the user it's return would be even worse :)

**Comment by SmArtKar** [comment]
uuuuuh and I can't figure out why CI errors

**Comment by MrMelbert** [comment]
You might be able to bump all the plant analyzer procs to global, like how health analyzers's `chemscan` and `healthscan` are. Just an idea - not sure if it's any better.

**Comment by BebeYoshi** [comment]
Now I can finally watch the gamer plants being done

File: code/modules/hydroponics/grafts.dm [hunk 1]
```diff
@@ -46,3 +46,21 @@
...
+			text += "- Graftable Traits: [span_notice("[stored_trait.get_name()]")] -\n"
+		text += "*---------*\n"
+		text += "- Yield: [span_notice("[yield]")]\n"
+		text += "- Production speed: [span_notice("[production]")]\n"
+		text += "- Endurance: [span_notice("[endurance]")]\n"
+		text += "- Lifespan: [span_notice("[lifespan]")]\n"
+		text += "- Weed Growth Rate: [span_notice("[weed_rate]")]\n"
+		text += "- Weed Vulnerability: [span_notice("[weed_chance]")]\n"
+		text += "*---------*</span>"
+		. += text
```
Comment (LemonInTheDark) [inline code/modules/hydroponics/grafts.dm:66]: Is this stuff copy pasted from plant analyzer handling? why not unify the behavior?

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**